src/ast.c

```diff
@@ -52,6 +52,9 @@ JL_DLLEXPORT jl_sym_t *jl_new_sym;
 JL_DLLEXPORT jl_sym_t *jl_using_sym;
 JL_DLLEXPORT jl_sym_t *jl_splatnew_sym;
 JL_DLLEXPORT jl_sym_t *jl_block_sym;
+JL_DLLEXPORT jl_sym_t *jl_if_sym;
+JL_DLLEXPORT jl_sym_t *jl_for_sym;
+JL_DLLEXPORT jl_sym_t *jl_while_sym;
```

Comment:
Looks like these are unused?

Reply:
oh, true, this was from an earlier version